app/components/web-components/au-page-size.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="auk-form-group-layout auk-form-group-layout--standard">
+  <div class="auk-form-group">
+    <PowerSelect
+          class="auk-select auk-select--block" id="selectItemsPerPage"
+          @searchEnabled={{false}}
+          @placeholder={{t "select-a-size"}}
```

Comment:
placeholder hebben we in principe niet nodig hier.
Er is geen search vak, en default selecteren we normaal al een waarde (10).
Dus we hoeven nooit een placeholder te tonen